identityClient/api.go

```diff
@@ -718,3 +718,19 @@ type ListRealmApplicationMappersRequest struct {
 type ListRealmApplicationMappersResponse struct {
 	ApplicationMappers []ApplicationMapper `json:"application_mappers"`
 }
+
+// SearchRealmIdentitiesResponse wraps matching identities in the given realm for the search criteria (if any)
+type SearchRealmIdentitiesResponse struct {
+	MatchingEmailIdentityAliases    SearchCriteriaToToznyClientAliases `json:"identities_email_to_client_aliases"`
+	MatchingUsernameIdentityAliases SearchCriteriaToToznyClientAliases `json:"identities_username_to_client_aliases"`
```

Comment:
We haven't really used alias in other places. Is this really an alias? Maybe `MatchingUsernameIdentities` and `identities_username_to_client_id` and something similar for email?

Reply:
Though this has likely all been debated and settled on in api-spec and identity service, so feel free to ignore

Reply:
While those are fantastic names for this current use case, my thinking is today the alias is a Tozny Client ID, but in the future it could be a group (TozId or TozStore) group id

Reply:
perfect, as a java guy I love when field names become very long